templates/fluent-resource/en-US/code-of-conduct.ftl

```diff
@@ -0,0 +1,113 @@
+### Translation file for page: https://www.rust-lang.org/policies/code-of-conduct
+
+## templates/policies/code-of-conduct.hbs
+
+coc-page-heading = Code of conduct
+
+coc-conduct-heading = Conduct
+coc-conduct-li1 =
```

Comment:
The entire HTML code of conduct should be a single key

Reply:
@Manishearth Just checking, but I'm assuming this doesn't apply to headings? There are two sections to the COC, and `h3` tags have ids on them. Does the whole page need to be stored as a unit for legal reasons?

Reply:
Depends on the case, but I would treat the moderation policy as a separate pair of keys (header + text)

Overall we should be very careful about complex formatting making its way in (like, class names, etc) but simple tags are fine.

This is a bit of an exercise in balance, in the case of a bajillion bullet items the whole thing should be a single key, but if there are just two it's fine to split.